### ISSUE TITLE
Beta FreeCodeCamp Challenges 

### ISSUE THREAD

**Issue by noone396** [freeCodeCamp/freeCodeCamp#13443]
I've been using the beta version of the new freecodecamp for a few days now and I am loving the updates. I think a good improvement would be to have the next challenge already loaded with the before results instead of having to click on the "Run test" before starting it.

**Comment by raisedadead** [comment]
@noone396 thanks for reporting. Closing in favor of older issue https://github.com/freeCodeCamp/freeCodeCamp/issues/12923
